src/renderer.js

```diff
@@ -484,6 +484,16 @@ EPUBJS.Renderer.prototype.firstElementisTextNode = function(node) {
 	return false;
 };
 
+EPUBJS.Renderer.prototype.isGoodNode = function(node) {
+	"use strict";
```

Comment:
Oh yeah, sorry, overlooked the global one :). I'll do a new pull request
without the use strict :)

2015-01-08 21:17 GMT+01:00 Fred Chasen <notifications@github.com>:

> In src/renderer.js
> <https://github.com/futurepress/epub.js/pull/183#discussion-diff-22678237>
> :
>
> > @@ -484,6 +484,16 @@ EPUBJS.Renderer.prototype.firstElementisTextNode = function(node) {
> >  	return false;
> >  };
> >
> > +EPUBJS.Renderer.prototype.isGoodNode = function(node) {
> > +	"use strict";
>
> I think we should avoid putting it in individual functions as it is
> already added here (
> https://github.com/futurepress/epub.js/blob/master/src/base.js#L1) and
> the files are concatenated when built.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/futurepress/epub.js/pull/183/files#r22678237>.
>